packages/contracts/test/token/erc20/governance-erc20.ts

```diff
@@ -374,5 +383,401 @@ describe('GovernanceERC20', function () {
 
       expect(await token.getVotes(signers[1].address)).to.equal(30);
     });
+
+    context('exhaustive tests', async () => {
+      context('`to` has a zero balance', async () => {
+        beforeEach(async () => {
+          expect(await token.balanceOf(to.address)).to.eq(0);
+          toDelegate = addressZero;
+        });
+
+        context('`to` delegated to `other`', async () => {
+          beforeEach(async () => {
+            await expect(token.connect(to).delegate(other.address))
+              .to.emit(token, 'DelegateChanged')
+              .withArgs(to.address, toDelegate, other.address);
+            toDelegate = other.address;
+          });
+
+          context('`to` receives via `mint` from `address(0)`', async () => {
+            beforeEach(async () => {
+              await expect(token.mint(to.address, 100)).to.not.emit(
+                token,
+                'DelegateChanged'
+              );
+            });
+
+            it('`to` has the correct voting power', async () => {
+              expect(await token.getVotes(to.address)).to.equal(0);
+            });
+            it('`to`s delegate has not changed', async () => {
+              expect(await token.delegates(to.address)).to.equal(toDelegate);
+            });
+            it('`to`s delegate has the correct voting power', async () => {
+              expect(await token.getVotes(toDelegate)).to.equal(100);
+            });
+          });
+
+          context('`to` receives via transfer from `from`', async () => {
+            beforeEach(async () => {
+              await token.mint(from.address, 100);
+              await expect(
+                token.connect(from).transfer(to.address, 100)
+              ).to.not.emit(token, 'DelegateChanged');
+            });
+
+            it('`from` has the correct voting power', async () => {
+              expect(await token.getVotes(to.address)).to.equal(0);
```

Comment:
check doesn't match the description

Reply:
Fixed in c0d3b11a35f333f3c0dac00e73c4f7f2edb89620